packages/documentation/cypress/snapshots/components/collapsible.snapshot.ts

```diff
@@ -2,6 +2,7 @@ describe('Collapsible', () => {
   it('default', () => {
     cy.visit('/iframe.html?id=snapshots--collapsible');
     cy.get('post-collapsible.hydrated', { timeout: 30000 }).should('be.visible');
+    cy.wait(500); // Wait for collapse animation to run
```

Comment:
Will you do another fix to hide the collapsible (I mean outside of the snapshot)? On the [docs page](https://preview-2946--swisspost-design-system-next.netlify.app/?path=/docs/6a91848c-16ec-4a23-bc45-51c797b5b2c3--docs), I see the border-top line when collapsed until I scroll on Chrome.

https://github.com/swisspost/design-system/assets/12294151/7303aa1b-91ea-4ab2-b40b-d540d241bd02